extensions/prometheus-grafana-k8s/v1/prometheus_values.yaml

```diff
@@ -163,7 +171,7 @@ configmapReload:
   ##
   image:
     repository: jimmidyson/configmap-reload
-    tag: v0.2.2
+    tag: v0.1
```

Comment:
Is there a reason not to use 0.2.2?
0.1 is from some time in 2016

Reply:
That's the default in the Prometheus helm chart v5.1.3. In my opinion, if we want to see this changed then it should be reflected on the charts repo, and then we can PR here to bump up the version of the chart itself. At this point, we want to stay as stable as possible by sticking with default chart values where appropriate.